src_bagl/ui_flow.c

```diff
@@ -48,75 +57,83 @@ UX_FLOW(ux_idle_flow,
         &ux_idle_flow_4_step,
         FLOW_LOOP);
 
-#if defined(TARGET_NANOS)
-
 // clang-format off
 UX_STEP_CB(
-    ux_settings_flow_1_step,
+    ux_settings_flow_blind_signing_step,
+#ifdef TARGET_NANOS
     bnnn_paging,
+#else
+    bnnn,
+#endif
     switch_settings_blind_signing(),
     {
+#ifdef TARGET_NANOS
       .title = "Blind signing",
-      .text = strings.common.fullAddress,
+      .text =
+#else
+      "Blind signing",
+      "Transaction",
+      "blind signing",
+#endif
+      strings.common.fullAddress
     });
 
 UX_STEP_CB(
-    ux_settings_flow_2_step,
+    ux_settings_flow_display_data_step,
+#ifdef TARGET_NANOS
     bnnn_paging,
+#else
+    bnnn,
+#endif
     switch_settings_display_data(),
     {
+#ifdef TARGET_NANOS
       .title = "Debug data",
-      .text = strings.common.fullAddress + 12
+      .text =
```

Comment:
It equals to the lines that come after, which vary depending on what the preprocessor produces.
And since the preprocessor directives have to have their own line, this line on its own doesn't make sense, but reading the following ones does.

This refactoring was meant to reduce the amount of duplicated code between different devices.